modules/calcite/src/test/java/org/apache/ignite/internal/processors/query/calcite/logical/SqlScriptRunner.java

```diff
@@ -714,7 +717,8 @@ private void checkResultsHashed(List<List<?>> res) {
 
             if (!res0.equals(expectedHash))
                 throw new AssertionError("Unexpected hash result, expected=" + expectedHash +
-                    ", values=" + res.size() * res.get(0).size() + ", expected=" + expectedRows);
+                    ", values=" + res.size() * res.get(0).size() + ", expected=" + expectedRows + ", calculated=" +
```

Comment:
Let's add also `posDesc` to the message for navigating from IDE in case of failure (see other messages as example)